styles.css

```diff
@@ -1,3 +1,345 @@
 body {
-  background-color: green;
+  background-color: lightgray;
+}
+
+a {
+  text-decoration: none;
+}
+
+.main-layout {
+  width: 400px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  background-color: white;
+  margin-left: auto;
+  margin-right: auto;
+  overflow: hidden;
+}
+
+nav {
+  height: 50px;
+  background-color: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: white;
+  width: 400px;
+  position: fixed;
+  margin: 0;
+}
+
+nav span {
+  margin: 0 5px;
+}
+
+nav img {
+  margin: 0 5px;
+}
+
+.intro-section {
+  background-color: #1c1a19;
+}
+
+.intro-section h1 {
+  color: #ff6b00;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  background-size: cover;
+  padding: 0 18px;
+}
+
+.intro-section h4 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  color: #fff5e1;
+  margin: 10px;
+  padding: 0 18px;
+}
+
+.intro-section p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  padding: 0 18px;
+}
+
+.contact-icons a {
+  margin: 20px 7px;
+}
+
+.contact-icons {
+  width: 100%;
+  margin: 0 15px 0 0;
+  height: 100px;
+}
+
+.wazzi {
+  margin-left: 10px;
+}
+
+.bg-handle {
+  background-image: url('./Assets/Group.png');
+  background-repeat: no-repeat;
+  background-position: bottom 300px;
+  background-position: right;
+  background-size: 100px 200px;
+  padding: 0;
+}
+
+.recent-work {
+  width: 80%;
+  font-size: 25px;
+  margin: 15px auto;
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.recent-work h4 {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 30px;
+  line-height: 52px;
+  margin: 20px;
+}
```

Comment:
- [ ] Kindly correct the font style, size, and weight of the title to be similar to the Figma template. 👇🏽 

- Figma template
![image](https://user-images.githubusercontent.com/58642949/187033633-6420ba84-7ccd-40f1-b378-a40db0873f3b.png)


- Your design
![image](https://user-images.githubusercontent.com/58642949/187033618-cd6f36e0-bf19-4970-992e-d4f1729d4277.png)